requirements-dev.txt

```diff
@@ -17,5 +16,5 @@ qiskit-aer
 mypy>=0.780
 mypy-extensions>=0.4.3
 nbsphinx
-qiskit_sphinx_theme>=1.10.*
+qiskit_sphinx_theme~=1.11.1
```

Comment:
This pin is more stable. I will open PRs to update to the new theme when we make releases.